pkg/apis/networking/v1/defaults.go

```diff
@@ -43,3 +46,12 @@ func SetDefaults_NetworkPolicy(obj *networkingv1.NetworkPolicy) {
 		}
 	}
 }
+
+func SetDefaults_IngressClass(obj *networkingv1.IngressClass) {
```

Comment:
minor: you could make this named and typed for the Params struct, so you don't need the additional nil-check (I think)

Reply:
Tried but this didn't work. I'm not familiar with codegen here but based on some reading of a few `zz_generated.defaults.go` that's not the case.
Furthermore, `scheme.AddTypeDefaultingFunc()` requires a type which implements `runtime.Object`.